Products/ZenUI3/browser/resources/js/zenoss/Reports.js

```diff
@@ -158,12 +158,19 @@ Ext.define('Zenoss.ReportTreePanel', {
             msg: message,
             fn: function(buttonid) {
                 if (buttonid==='ok') {
+                    report_tree._deleteChilds();
                     report_tree._deleteSelectedNode();
                 }
             },
             buttons: Ext.MessageBox.OKCANCEL
         });
     },
+    _deleteChilds: function () {
```

Comment:
Please name this function `_deleteChildren` (or `_deleteSubNodes`), i.e. 'children' is the plural form of 'child'.